commit 31293ae7074802cc7286089a89c7b552d930c97f
Author: Jia Tan <jiat0218@gmail.com>
Date:   Tue Jan 9 16:40:56 2024 +0800

    Doxygen: Added the XZ logo and copyright information.
    
    The PROJECT_LOGO field is now used to include the XZ logo. The footer
    of each page now lists the copyright information instead of the default
    footer. The license is also copied to statisfy the copyright and so the
    link in the documentation can be local.

diff --git a/doc/logo/copyright.html b/doc/logo/copyright.html
new file mode 100644
index 00000000..2c687c56
--- /dev/null
+++ b/doc/logo/copyright.html
@@ -0,0 +1,11 @@
+<hr class="footer"/>
+
+<p style="text-align: right;padding-right: 12px;">
+	XZ logo &copy; 2023 by Jia Tan is licensed under
+	<a href="COPYING.CC-BY-SA-4.0"
+	   target="_blank"
+	   rel="license noopener noreferrer"
+	   style="display:inline-block;">
+		CC BY-SA 4.0
+	</a>
+</p>
diff --git a/doc/logo/xz-logo.png b/doc/logo/xz-logo.png
new file mode 100644
index 00000000..f9d6313d
Binary files /dev/null and b/doc/logo/xz-logo.png differ
diff --git a/doxygen/Doxyfile b/doxygen/Doxyfile
index 14350cf4..a4bc9b8d 100644
--- a/doxygen/Doxyfile
+++ b/doxygen/Doxyfile
@@ -61,7 +61,7 @@ PROJECT_BRIEF          =
 # pixels and the maximum width should not exceed 200 pixels. Doxygen will copy
 # the logo to the output directory.
 
-PROJECT_LOGO           =
+PROJECT_LOGO           = ../doc/logo/xz-logo.png
 
 # The OUTPUT_DIRECTORY tag is used to specify the (relative or absolute) path
 # into which the generated documentation will be written. If a relative path is
@@ -1246,7 +1246,7 @@ HTML_HEADER            =
 # that doxygen normally uses.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_FOOTER            =
+HTML_FOOTER            = ../doc/logo/copyright.html
 
 # The HTML_STYLESHEET tag can be used to specify a user-defined cascading style
 # sheet that is used by each HTML page. It can be used to fine-tune the look of
@@ -1286,7 +1286,7 @@ HTML_EXTRA_STYLESHEET  =
 # files will be copied as-is; there are no commands or markers available.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_EXTRA_FILES       =
+HTML_EXTRA_FILES       = ../COPYING.CC-BY-SA-4.0
 
 # The HTML_COLORSTYLE tag can be used to specify if the generated HTML output
 # should be rendered with a dark or light theme.